addons/core/functions/fnc_initKeybinds.sqf

```diff
@@ -114,56 +114,63 @@ TF_speak_volume_modifier_yelling_modifiers = [false, false, false];
-["TFAR","SWStereoRight",["SW Stereo: Right","SW Stereo: Right"],{[2] call TFAR_fnc_processSWStereoKeys},{true},[TF_sw_stereo_right_scancode,TF_sw_stereo_right_modifiers],false] call cba_fnc_addKeybind;
-
-
-["TFAR","LRStereoBoth",    ["LR Stereo: Both","LR Stereo: Both"],{[0] call TFAR_fnc_processLRStereoKeys},    {true},[TF_lr_stereo_both_scancode, TF_lr_stereo_both_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","LRStereoLeft",    ["LR Stereo: Left","LR Stereo: Left"],{[1] call TFAR_fnc_processLRStereoKeys},    {true},[TF_lr_stereo_left_scancode, TF_lr_stereo_left_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","LRStereoRight",    ["LR Stereo: Right","LR Stereo: Right"],{[2] call TFAR_fnc_processLRStereoKeys},{true},[TF_lr_stereo_right_scancode,TF_lr_stereo_right_modifiers],false] call cba_fnc_addKeybind;
-
-
-["TFAR","SWTransmit",["SW Transmit","SW Transmit"],{call TFAR_fnc_onSwTangentPressed},{call TFAR_fnc_onSwTangentReleased},[TF_tangent_sw_scancode, TF_tangent_sw_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","SWTransmitAlt",["SW Transmit Alt","SW Transmit Alt"],{call TFAR_fnc_onSwTangentPressed},{call TFAR_fnc_onSwTangentReleased},[TF_tangent_sw_2_scancode, TF_tangent_sw_2_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","SWTransmitAdditional",["SW Transmit Additional","SW Transmit Additional"],{call TFAR_fnc_onAdditionalSwTangentPressed},{call TFAR_fnc_onAdditionalSwTangentReleased},[TF_tangent_additional_sw_scancode, TF_tangent_additional_sw_modifiers],false] call cba_fnc_addKeybind;
-
-["TFAR","LRTransmit",["LR Transmit","LR Transmit"],{call TFAR_fnc_onLRTangentPressed},{call TFAR_fnc_onLRTangentReleased},[TF_tangent_lr_scancode, TF_tangent_lr_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","LRTransmitAlt",["LR Transmit Alt","LR Transmit Alt"],{call TFAR_fnc_onLRTangentPressed},{call TFAR_fnc_onLRTangentReleased},[TF_tangent_lr_2_scancode, TF_tangent_lr_2_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","LRTransmitAdditional",["LR Transmit Additional","LR Transmit Additional"],{call TFAR_fnc_onAdditionalLRTangentPressed},{call TFAR_fnc_onAdditionalLRTangentReleased},[TF_tangent_additional_lr_scancode, TF_tangent_additional_lr_modifiers],false] call cba_fnc_addKeybind;
-
-
-["TFAR","LowerHeadset",["Lower Headset","Lower Headset"],{},{true call TFAR_fnc_setHeadsetLowered;},[0, [false, false, false]],false] call cba_fnc_addKeybind;
-["TFAR","RaiseHeadset",["Raise Headset","Raise Headset"],{},{false call TFAR_fnc_setHeadsetLowered;},[0, [false, false, false]],false] call cba_fnc_addKeybind;
+_fnc_localizeSWChannel = {
+  private _str = format ["%1 %2",localize "STR_TFAR_key_SW_Channel", _this];
+  [_str, _str]
+};
+["TFAR","SWChannel1",LOCALIZE_CHANNEL(SW,1),{[0] call TFAR_fnc_processSWChannelKeys},{true},[TF_sw_channel_1_scancode, TF_sw_channel_1_modifiers],false] call cba_fnc_addKeybind;
```

Comment:
@dedmen 
I call marcro here.

Reply:
Jup

Reply:
Is the tooltipp even needed, if it is the same like the name?
and:
```
#define LOCALIZE_CHANNEL(RANGE,NUMBER) [ARR_2(localize LSTRING(TRIPLES(DOUBLES(KEY,RANGE),CHANNEL,NUMBER)),localize LSTRING(TRIPLES(DOUBLES(KEY,RANGE),CHANNEL,NUMBER)))]
```

Reply:
No. But doesn't really hurt I guess.
Also if people are at some ridiculous font size they might not be able to see the full name